src/components/Network.vue

```diff
@@ -6,20 +6,20 @@
         <button class="btn btn-primary btn-sm ml-auto">Update Info</button>
       </div>
       <div class="row pt-4">
-        <div class="col-md-7 border-right">
+        <div class="col-md-7">
+          <h5 class="text-left pb-2">Graph</h5>
+          <NetworkGraph :graphName="'Simulation Environment'" />
+        </div>
+        <div class="col-md-5 border-right">
```

Comment:
```suggestion
        <div class="col-md-5">
 ```
めちゃくちゃどうでもいいことなんですが、ここの線だけ消したい感があります！